public/script.js

```diff
@@ -1720,16 +1720,24 @@ export async function replaceCurrentChat() {
 }
 
 export function showMoreMessages() {
-    let messageId = Number($('#chat').children('.mes').first().attr('mesid'));
+    const firstDisplayedMesId = $('#chat').children('.mes').first().attr('mesid');
+    let messageId = Number(firstDisplayedMesId);
     let count = power_user.chat_truncation || Number.MAX_SAFE_INTEGER;
 
+    // If there are no messages displayed, or the message somehow has no mesid, we default to one higher than last message id,
+    // so the first "new" message being shown will be the last available message
+    if (!messageId) {
```

Comment:
Possibly. I mean, it's nearly the same. And in that case it doesn't matter that it can be 0, because in that case we cannot prepend anyway, so both work...
![image](https://github.com/user-attachments/assets/0357802f-5f57-4a64-958e-68c6243b3e21)

But being more direct in intention never hurts.

(see, that what it leads to when you do JS devving. Over-reliance on *eh, the language says this is falsey anyway, so just go with it xD )